actions/actions.py

```diff
@@ -0,0 +1,163 @@
+from typing import Any, Text, Dict, List
+import json
+import requests
+import sqlite3
+import textwrap
+
+
+from rasa_sdk import Action, Tracker
+from rasa_sdk.events import UserUtteranceReverted
+from rasa_sdk.executor import CollectingDispatcher
+
+
+class ActionDefaultFallback(Action):
+    """Executes the fallback action and goes back to the previous state
+    of the dialogue"""
+
+    customer_id = "1835841754"
+    corpus_id = 1
```

Comment:
Is this a valid corpus_id? Does the API key and customer id work as-is? We want to use valid values here so that people can try out this particular example without creating their own account on our platform.

Reply:
Yes, it is a valid corpus id linking to the prod.

Reply:
Its the first corpus created from the production account Tallat created.

Reply:
Okay. The new account Tallat created is just a playground. We'll want to put this dataset into our production demos account. Can you please share the documents with me and I'll create a corpus + API key for you.